### PR TITLE
Add DEI Mission Statement

### LINKED ISSUE #54: DEI Statement in higher level menu
Once the new DEI statement (https://us-rse.org/dei-mission/) is brought over to the new site, we should feature it properly. I assume that would mean a high level menu item in "about" but could be elsewhere.

### PR DESCRIPTION
**Pull request by cosden**
This adds:

- A new page with the DEI Mission Statement
- An "About" menu item "DEI Mission Statement" that comes immediately after "Mission"

It's a direct copy from the old website with one change. Because we aren't displaying the subtitle field, I added it as a separate heading. I thought it worth keeping because it spells out DEI. 

Preview:
image

This closes #54.

### CODE REVIEW

File: pages/about/dei-mission-statement.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+religion (or lack thereof), national origin, and ethnicity.
+
+The US-RSE Association will:
+- Continually educate ourselves through research on best practices focused on
+  diversity, equity, and inclusion
+- Embrace the culture and diversity of all our members
+- Welcome and respect ways of thinking and learning from people of all
+  backgrounds
+- Seek to make our membership and the RSE community in the US more broadly
+  representative of  our national population
```
Comment (mtbc) [inline pages/about/dei-mission-statement.md:29]:
```suggestion
  representative of our national population
```

**Comment by cmaimone** [comment]
I would suggest addressing the page.html layout instead of just adding in the subtitle manually.  For the work underway on the formatting of the news, I had to add subtitle to the post.html layout.  Any reason not to change the layout?

**Comment by vsoch** [comment]
@cmaimone I disagree - not all pages have subtitles and it's super easy to just do `## subtitle thing` (or whatever other format is wanted in the markdown). The page automatically handles the title, and I think the rest should be up to the specific page in question. If it's discovered that, for example, an event (or similar) needs a common layout, then it makes sense to make a custom page. But page.html should be sort of a vanilla base for a generic page, meaning not biases to have a particular subtitle or any other structure beyond a title.

**Comment by vsoch** [comment]
Also - if you edited post.html that is also going to change the blog posts, which I don't think we want to necessarily do (do we think a news post should look different than a blog post or not?) So the solution to that (if they should be different) might be to make  a custom news post page base.

**Review by vsoch — APPROVED** [review]

**Comment by cosden** [comment]
This may be worth a discussion at our next meeting, but my initial reaction is the same as @vsoch. Using markdown formatting is easy and more flexible. The old site had things set up in a way that it would have been hard to replicate just with markdown, but the new layout doesn't have that problem. @cmaimone if you think we should discuss it at a meeting, feel free to add it as an issue.

**Pull request merged**